### PR TITLE
add new local icon option

### PR DESCRIPTION
**Pull request by francnc**
### What

Add the option for providing an SVG icon as a prop so we don't have to reference the icons from our libraries as the only way of showing icons in cwRadioInputs

### How

Adding a new property to the component and rendering it when passed as a prop

### CODE REVIEW

File: lib/components/IconRadioInput.jsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ class IconRadioInput extends React.Component {
       iconClass,
       recommendable,
       className,
+      localIcon,
     } = this.props;
     const { checked } = this.state;
     const buttonClass = classNames(
```
Comment (jgsojo) [inline lib/components/IconRadioInput.jsx]: Probably we need to update condition to set 'icons-radio-input--no-icon' class.

File: lib/components/IconRadioInput.jsx [hunk 2]
```diff
@@ -56,7 +57,13 @@ class IconRadioInput extends React.Component {
     );
     const inputId = `${name}${value}`;
     return (
+      
       <label className={buttonClass} htmlFor={inputId}>
+        {localIcon && (
```
[the remark this answers is not in the document]
Comment (francnc) [reply lib/components/IconRadioInput.jsx]: You're right, I'm moving it!

File: lib/components/IconRadioInput.jsx [hunk 3]
```diff
@@ -42,6 +42,7 @@ class IconRadioInput extends React.Component {
       iconClass,
       recommendable,
       className,
+      localIcon,
```
[the remark this answers is not in the document]
Comment (francnc) [reply lib/components/IconRadioInput.jsx]: Yes, `icon` seems better naming, I'll add it to the propTypes.

File: lib/components/IconRadioInput.jsx (hunk 1 above)
Comment (francnc) [reply lib/components/IconRadioInput.jsx]: yes, we need to check also if `icon` exists

**Pull request merged**